GothicVRProject/GothicVR
# Feature/270 root motion + Routines for WPs + grounded walk

# To test
1. Set FeatureFlag.spawnNPcs = 233 (Bloodwyn)
2. Set FeatureFlag.time = 8:55
3. Check if he walks to a FreePoint, waiting for a few seconds, then walking to a path to the other side of the camp

Boundary:
* Walk is still a little bit wonky, but Bloodwyn shouldn't fall through the ground.
* A lot of changes are due to rework of FeatureFlags

# Checklist

## Scene

# Testing

## Dependencies
